### ISSUE TITLE
[gazellegames] Your cookie did not work: Parse error (Config)

### ISSUE THREAD

**Issue by psylk** [Jackett/Jackett#2555]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue. Duplicating issues makes it more difficult for everyone to follow. Your issue may have already been solved in the past as well.

Provide a description of the feature request or bug, the more details the better.
If you are experiencing an issue with a tracker, a **full log must be included**. Instructions for obtaining logs are here: https://github.com/Jackett/Jackett#troubleshooting

**Jackett version**:  0.8.670.0
**Mono version** (if not using Windows):  5.4.1.6

-------------------------------

The cookie seems to be formatted the same as AHD, which works fine.
I've deleted GGN cookies in chrome and re-logged in, which purged some older cookies, still won't work.

It seems from the docker logs, it's logged in and it's dumped the torrents page with the empty search.
    `Debug WebClient(HttpWebClient): Returning OK => [ggn torrents page html]`
After that I can see 2 errors
    `Error ReflectedHttpActionDescriptor ExecuteAsync`
    `Error ApiControllerActionInvoker InvokeActionAsync`

And getting a bunch(28) of stack traces (in reverse order from logs)
```
--- End of stack trace from previous location where exception was thrown ---
at Jackett.Controllers.V20.IndexerApiController+<UpdateConfig>d__10.MoveNext () [0x00118] in <007bc804053d400eb5a797187d5e6245>:0
at System.Runtime.CompilerServices.TaskAwaiter`1[TResult].GetResult () [0x00000] in <62f5937022004555807e6c57c33f6684>:0
at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) [0x00008] in <62f5937022004555807e6c57c33f6684>:0
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) [0x00028] in <62f5937022004555807e6c57c33f6684>:0
at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) [0x0003e] in <62f5937022004555807e6c57c33f6684>:0
at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <62f5937022004555807e6c57c33f6684>:0
```
And that's about as far as I can get reading C#

**Comment by kaso17** [comment]
Please share the HTML from the log (but make sure to remove the authkey/passkey, etc. details)

**Comment by psylk** [comment]
My bad, here's the actual log and html, not reversed.

[jackett-ggn.log](https://github.com/Jackett/Jackett/files/1703648/jackett-ggn.log)
[jackett-ggn-html.txt](https://github.com/Jackett/Jackett/files/1703652/jackett-ggn-html.txt)

**Comment by kaso17** [comment]
Fixed via da51c070d0ca72f589a3eee69e04e9ee3f1b28c2
For some reason you have an extra column (the last one) which I don't have.
